modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/edit_entry.jsp

```diff
@@ -139,10 +139,9 @@ renderResponse.setTitle((entry != null) ? BlogsEntryUtil.getDisplayTitle(resourc
 							<small>
 								<liferay-editor:editor
 									contents="<%= coverImageCaption %>"
-									editorName="alloyeditor"
+									editorName="ballooneditor"
 									name="coverImageCaptionEditor"
 									placeholder="caption"
-									showSource="<%= false %>"
```

Comment:
We cannot just temporarily remove a feature from product team component. We can, if there is a really good reason for it, but there is none in this case. We need to implement the [source mode support](https://issues.liferay.com/browse/LPS-137763) first.